### PR TITLE
Added rake and travis ci file

### PR DESCRIPTION
**Pull request by jonathanshum**
Rake + .travis.yml

Might need to edit config/cucumber.yml to stop email notifications

Reviewer: @DarryQueen

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -14,6 +14,9 @@ group :development, :test do
   gem 'simplecov'
 end
 
+# Travis CI
```
Comment (DarryQueen) [inline Gemfile]: You should edit your comment. Rake is not exclusive to Travis CI.

Also make sure to run `bundle install` after editing Gemfile, so Gemfile.lock gets updated as well.

**Comment by DarryQueen** [comment]
See line notes.

**Comment by jonathanshum** [comment]
Travis CI is set up. Need keys for facebook & google for the tests to pass.

**Comment by DarryQueen** [comment]
We won't be posting Facebook or Google keys publicly, so there has to be a workaround.

Let's refrain from merging until we're sure TravisCI won't give us stupid errors.

**Comment by jonathanshum** [comment]
Looks like Travis CI is all set up and builds are passing now!

Reviewer: @DarryQueen

**Comment by DarryQueen** [comment]
Not sure if we want me to review this since I was a collaborator. Passing on to Naseem.

Reviewer: @nalnaji

**Comment by nalnaji** [comment]
Nice, it works well. LGTM.

**Pull request merged**